dataprofiler/data_readers/avro_data.py

```diff
@@ -67,7 +67,7 @@ def _load_data_from_file(self, input_file_path: str) -> List:
             # even when the option encoding='utf-8' is added. It may come from
             # some special compression codec, e.g., snappy. Then, binary mode
             # reading is currently used to get the dict-formatted lines.
-            df_reader: fastavro.reader = fastavro.reader(input_file)
+            df_reader = fastavro.reader(input_file)
```

Comment:
was this unnecessary?

Reply:
it was actually throwing a `mypy` error IIRC @menglinw